src/main/row.ts

```diff
@@ -455,6 +472,202 @@ function getHelperGroupFromHelperList(helperNameList: string, role: string | und
+    const allHelpers: string[] = [];
+    const helperLines: string[] = getHelperLines(row);
+
+    for(const helperLine of helperLines) {
+        if (hasRole(helperLine)) {
+            const [role, helperList] = helperLine.split(COLON_DELIM);
+            const trimmedHelperList: string = helperList.trim();
+            allHelpers.push(...getHelpersNames(trimmedHelperList));
+            
+        } else if(helperLine !== "") {
+            allHelpers.push(...getHelpersNames(helperLine));
+        }
+    }
+
+    return allHelpers;
+}
+
+/**
+ * Retrieves an array of domain names from a row
+ * 
+ * @param row row to retrieve domain names from
+ * @returns array of domain names
+ */
+function getDomainNames(row: Row): string[] {
+    return row.domain.split(COMMA_FORWARD_SLASH_DELIM_REGEX)
+    .map(value => value.trim()).filter(value => GROUP_NAMES.includes(value));
+}
+
+/**
+ * Retrieves an array of domain filters from a row
+ * 
+ * @param row row to retrieve domain filters from
+ * @returns array of domain filters
+ */
+function getDomainFilters(row: Row): string[] {
+    return row.domain.split(COMMA_FORWARD_SLASH_DELIM_REGEX)
+    .map(value => value.trim()).filter(value => isFilter(value));
+}
+
+/**
+ * Splits the value of the who column of a row based on the COMMA_FORWARD_SLASH_DELIM_REGEX
+ * 
+ * @param row - An event row
+ * @returns a split string representing who is attending an event
+ */
+function splitWhoColumn(row: Row): string[] {
+    return row.who.split(COMMA_FORWARD_SLASH_DELIM_REGEX);
+}
+
+/**
+ * Retrieves an array of ministry names from a row
+ * 
+ * @param row row to retrieve ministry names from
+ * @returns array of ministry group names
+ */
+function getMinistryNames(row: Row): string[] {
+    return splitWhoColumn(row)
+        .map(name => name.trim()).filter(value => GROUP_NAMES.includes(value.toUpperCase()));
+}
+
+/**
+ * Retrieves an array of ministry filters from a row
+ * 
+ * @param row row to retrieve ministry filters from
+ * @returns array of ministry filters
+ */
+function getMinistryFilters(row: Row): string[] {
+    return splitWhoColumn(row)
+    .map(name => name.trim()).filter(value => isFilter(value));;
+}
+
+/**
+ * Checks if the event has a who (ministy) column value of ROTATION
+ * 
+ * @param row - An event row
+ * @returns true or false
+ */
+export function CheckForRotation(row: Row): boolean {
+    return splitWhoColumn(row)[0].toUpperCase() === "ROTATION";
+}
+
+/**
+ * Checks if the event has a who (ministry) column value of VARIOUS
+ * 
+ * @param row - An event row
+ * @returns true or false
+ */
+export function CheckForVarious(row: Row): boolean {
+    return splitWhoColumn(row)[0].toUpperCase() ===  "VARIOUS";
+}
+
+/**
+ * Checks if the event has a who (ministry) column value of OTHER
+ * 
+ * @param row - An event row
+ * @returns true or false
+ */
+export function CheckForOther(row: Row): boolean {
+    return splitWhoColumn(row)[0].toUpperCase() ===  "OTHER";
+}
```

Comment:
```suggestion
export function checkForRotation(row: Row): boolean {
    return splitWhoColumn(row)[0].toUpperCase() === "ROTATION";
}

/**
 * Checks if the event has a who (ministry) column value of VARIOUS
 * 
 * @param row - An event row
 * @returns true or false
 */
export function checkForVarious(row: Row): boolean {
    return splitWhoColumn(row)[0].toUpperCase() ===  "VARIOUS";
}

/**
 * Checks if the event has a who (ministry) column value of OTHER
 * 
 * @param row - An event row
 * @returns true or false
 */
export function checkForOther(row: Row): boolean {
    return splitWhoColumn(row)[0].toUpperCase() ===  "OTHER";
}
```
Can we use camel case for these?